tsa-core/src/main/kotlin/org/usvm/machine/interpreter/TvmArtificialInstInterpreter.kt

```diff
@@ -419,10 +468,10 @@ class TvmArtificialInstInterpreter(
             unprocessedMessages = unprocessedMessages.addAll(newUnprocessedMessages.map { currentContract to it })
         }
 
-        return Unit
+        return messageDestinations.map { it.second }
```

Comment:
This returns only messages that will be accepted by other analyzed contracts. We should probably call `on_out_messages` for all messages, not just these.